static/app/main.tsx

```diff
@@ -20,7 +24,7 @@ function renderRouter(props: any) {
   );
 }
 
-const queryClient = new QueryClient();
+const queryClient = new QueryClient(DEFAULT_QUERY_CLIENT_CONFIG);
```

Comment:
since this is going to probably be the only place we are using `queryClient` do we need to 're-export' it in `utils/reactQuery`?

Reply:
Made this an export to ensure that the options don't diverge from the ones in the test environment - here's the discussion above that made me do it this way: https://github.com/getsentry/sentry/pull/40433#discussion_r1003594181

Reply:
in this case, should we also add `react-query` to the `no-restricted-imports`  list, like in the PR  https://github.com/getsentry/eslint-config-sentry/pull/129/files